### DISCUSSION TITLE
Where do "other notes" go after the inbox?

### DISCUSSION THREAD

**Discussion by bsoist** [BryanJenksCommunity/FAQ#89] [Q&A] [answered]
I asked a follow-up question over [here](https://github.com/BryanJenksCommunity/FAQ/discussions/27) but what I was really asking was about the statuses.

I know the final note in your example ends up with a status of evergreen note, but what do you do with the other two notes. I assume from your answer there that you keep them "separate from the slip box" but how - surely they don't say in the inbox, right? 

I think I found the literature note mentioned in that answer in your published notes, and I don't see any status at all. Do you simply remove the statuses?

**Comment by tallguyjenks** [comment, marked as answer]
the statuses are in the YAML front matter so they dont show on publish unless i turned on a setting for it, but they're just in the vault with their status that i mentioned in #27 so everything is in the vault at the root level nothing stays in the inbox once its completed

**Reply by bsoist** [reply]
I figured (YAML), but I just re-read #27 and I still have the same question, so I'll clarify...

I assume you use the daily notes like I do so that note doesn't have a status - no problem there. The third note becomes an evergreen note - understood.  But, I'm assuming your literature note starts in the inbox and I'm wondering what status you give it when you're done to remove it OR if you simply delete the status altogether.

**Reply by tallguyjenks** [reply]
there's no inbox for the literature notes. they move through stages of processing like a book getting processed and that is shown via tags that look like this: 

📥️/📚️/🟥️
📥️/📚️/🟧️
📥️/📚️/🟨️
📥️/📚️/🟩️

these are documented on the [tag taxonomy](https://publish.obsidian.md/bryan-jenks/Tag+Taxonomy) note I linked in #27.

no part of this process uses an inbox, its all done with links and tags. The literature notes in the end are left with a 📥️/📚️/🟩️ tag, no tags/statuses get deleted.

does that answer your question? :)

**Reply by bsoist** [reply]
It does, but I was interpreting your 📥️ as an inbox and I've been using it that way. 

I noticed you had a marker for literature notes under "graph groupings" so I started just labeling literature notes I'm done with a 🕸/🟪

If I don't think of 📥️ as an inbox, but as 📥️/📚/🟥 and 📥️/🎥/🟥 etc. as different "inboxes" that might work better for me anyway for a lot of reasons.

Thanks!
